### DISCUSSION TITLE
Can't import flet [Question]

### DISCUSSION THREAD

**Discussion by salexa2** [flet-dev/flet#1214] [Q&A]
### Question

Hi, I've been trying to import flet for hours. I'm using visual studio community for a group project and I have flet install but when i try to run my program to check if it's imported it says a bunch of modules can't be found like h2, socks, etc. It seems to be working on visual studio code but me and my group mates use visual studio community because it's easier to push our changes to git with it. Even when i manually try to install the missing modules it gives an error for something with attributes sending me back to square one. 

Shouldn't these modules be in flet if they're needed to import it. My program runs just find up until i add the import statement for flet. 

![image](https://user-images.githubusercontent.com/124205990/227671440-34fa8f5c-d448-4398-a620-89585b4387db.png)
![image](https://user-images.githubusercontent.com/124205990/227671607-8e1f697e-c9df-4f17-b77f-4ae0faeaa517.png)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Make sure *your* program is not called `flet.py`.
